src/gtp_path.erl

```diff
@@ -226,8 +226,9 @@ init([#gtp_port{name = PortName} = GtpPort, Version, RemoteIP, Args]) ->
 handle_event(enter, #state{peer = Old}, #state{peer = Peer}, Data)
   when Old /= Peer ->
     peer_state_change(Old, Peer, Data),
-    State = peer_state(Peer),
-    {keep_state_and_data, enter_peer_state_action(State, Data)};
+    OldState = peer_state(Old),
+    NewState = peer_state(Peer),
+    {keep_state_and_data, enter_peer_state_action(OldState, NewState, Data)};
```

Comment:
The `peer_state/1` will return atoms `down/idle/busy` and does nothing else. Maybe make sense clean up the code, remove call `peer_state/1` and also remove function `enter_peer_state_action/3` because looks like we always will return `[]`?